projects/elife.yaml

```diff
@@ -1693,6 +1693,32 @@ elife-libero-reviewer:
                 "{instance}-elife-libero-reviewer":
                     encryption: arn:aws:kms:us-east-1:512686554592:key/b6f44e77-46d8-41bb-a2e5-75c0588a2b20
 
+elife-libero-editor:
```

Comment:
the 'elife' prefix isn't necessary. I think 'libero-editor' is fine.

Reply:
I see 'elife-libero-reviewer' is already in there. You might want to keep it for consistency, I don't know. It just isn't necessary.

Reply:
I remember some other developers coming along once and thinking the 'elife-' prefix was some sort of company policy. It really isn't ;)